ts/src/flexible-event/privacy.test.ts

```diff
@@ -101,6 +102,90 @@ void test('maxInformationGain', async (t) => {
   )
 })
 
+void test('epsilonToBoundInfoGainAndDp', async (t) => {
+  const infoGainUppers = [11.5, 6.5]
+  const epsilonUpper = 14
+  const numStatesRange = 100000
+  const numTests = 500
+
+  await Promise.all(
+    [...Array(numTests).keys()].map((i) =>
```

Comment:
```suggestion
    Array(numTests).map((, i) =>
```

Reply:
Done.

Reply:
I reverted this because just using the map without filling the array did not seem to work. Either current code or, for example `...fill(0).map` worked.

Reply:
I forgot that [the `Array(length)` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Array#arraylength) is strange in that regard.

There's no need to use the `[...]` syntax with `keys()`, though, since that creates an intermediate array only to immediately create a new one from it. Just do:

```js
Array(numTests).fill(0).map((, i) => t.test(...)
```

Reply:
Done.